cg/meta/clean/flow_cell_run_directories.py

```diff
@@ -0,0 +1,88 @@
+"""Module that handles deletion of flow cell run directories and their BCL files from
+flow_cell_run_dir/<sequencer> """
+import logging
+import shutil
+from datetime import datetime, timedelta
+from pathlib import Path
+from typing import Optional
+
+from housekeeper.store import models as hk_models
+
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.constants import HousekeeperTags
+from cg.constants.demultiplexing import DemultiplexingDirsAndFiles
+from cg.constants.symbols import UNDERSCORE
+from cg.store import Store
+
+FLOW_CELL_DATE_POSITION = 0
+FLOW_CELL_IDENTIFIER_POSITION = 3
+LOG = logging.getLogger(__name__)
+
+
+class RunDirFlowCell:
+    """Class to check and remove flow cell run directories. If a flow cell has been sequenced 21
+    days ago or more, it is removed and the sample sheet is archived"""
```

Comment:
```suggestion
    """Class to check and remove flow cell run directories"""
```
Less is more. I do not think this class controls the 21 days.